pkg/asset/agent/installconfig_test.go

```diff
@@ -40,26 +45,53 @@ pullSecret: "{\"auths\":{\"example.com\":{\"auth\":\"authorization value\"}}}"
 			expectedError: `invalid install-config configuration: Platform: Unsupported value: "aws": supported values: "baremetal", "vsphere", "none"`,
 		},
 		{
-			name: "apiVips not set for baremetal platform",
+			name: "apiVips not set for baremetal Compact platform",
 			data: `
 apiVersion: v1
 metadata:
   name: test-cluster
 baseDomain: test-domain
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 
+    hostPrefix: 23 
+  networkType: OpenShiftSDN
+  machineNetwork:
+  - cidr: 192.168.122.0/23
+  serviceNetwork: 
+  - 172.30.0.0/16
+compute:
+  - architecture: amd64
+    hyperthreading: Enabled
+    name: worker
+    platform: {}
+    replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
 platform:
   baremetal:
+    externalMACAddress: "52:54:00:f6:b4:02"
+    provisioningMACAddress: "52:54:00:6e:3b:02"
```

Comment:
From `pkg/types/baremetal/platform.go`
```// ExternalMACAddress is used to allow setting a static unicast MAC
	// address for the bootstrap host on the external network. Consider
	// using the QEMU vendor prefix `52:54:00`. If left blank, libvirt will
	// generate one for you.
	// +optional
	ExternalMACAddress string `json:"externalMACAddress,omitempty"`
```
and
```
// ProvisioningMACAddress is used to allow setting a static unicast MAC
	// address for the bootstrap host on the provisioning network. Consider
	// using the QEMU vendor prefix `52:54:00`. If left blank, libvirt will
	// generate one for you.
	// +optional
	ProvisioningMACAddress string `json:"provisioningMACAddress,omitempty"`
````
Hence added in the test case, otherwise each run of the test case will generate a random macaddress